src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int numberCount = 100;
```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;
```
constants should be in uppercase